gr-zeromq/lib/base_impl.cc

```diff
@@ -31,8 +31,8 @@
 namespace gr {
   namespace zeromq {
 
-    base_impl::base_impl(int type, size_t itemsize, size_t vlen, int timeout, bool pass_tags)
-      : d_vsize(itemsize * vlen), d_timeout(timeout), d_pass_tags(pass_tags)
+    base_impl::base_impl(size_t type, size_t itemsize, size_t vlen, int timeout, bool pass_tags)
```

Comment:
I thought it was inconsistently declared between these classes, but now it looks like maybe I was wrong and was looking at the wrong variable.